src/index.ts

```diff
@@ -1,6 +1,6 @@
 import chalk from 'chalk';
+import child_process from 'child_process';
```

Comment:
Consider importing `execSync` here instead of the entire `child_process` module

```suggestion
import { execSync } from 'child_process';
```